core/src/i18n/en.yaml

```
@@ -962,6 +941,34 @@ jobs:
     status: Status
     title: Conditions
     type: Type
+  create-modal:
+    at-least-one-container-required: CronJob must have at least one container.
+    container: Container
+    containers: Containers
+    labels:
+      args: Args
+      command: Command
+      docker-image: Docker Image
+      failed-jobs-history-limit: Failed Jobs History Limit
+      image-pull-policy: Image Pull Policy
+      parallelism: Parallelism
+      starting-deadline: Starting Deadline
+      successful-jobs-history-limit: Successful Jobs History Limit
+      suspend: Suspend
+    placeholders:
+      args: Arguments to the entrypoint.
+      command: Command to run in a container.
+      docker-image: Enter the Docker image tag, for example, 'busybox'.
+      failed-jobs-history-limit: 'The number of failed finished jobs to retain (default: 1).'
+      parallelism: 'The maximum number of pods the job should run at any given time (default: 1).'
+      starting-deadline: Optional deadline in seconds.
+      successful-jobs-history-limit: 'The number of successful finished jobs to retain (default: 3).'
+    tooltips:
+      args: Docker image CMD is used if arguments are not provided. Provide either args or command.
+      command: The docker image's ENTRYPOINT is used if this is not provided. Provide either args or command.
```

```suggestion
      command: The Docker image's ENTRYPOINT is used if this is not provided. Provide either args or command.
```
Should the ENTRYPOINT be capitalized? It's not written that way everywhere in the doc.

---

I think it's Docker specific concept and it is capitalized in the Docker docs.